### PR TITLE
master lpd 27639

### PR DESCRIPTION
**Pull request by kyle-miho**
https://liferay.atlassian.net/browse/LPD-27639

cc: @nicolas-sss @drewbrokke

### CODE REVIEW

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:sf

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -803,7 +811,7 @@
 			"DefaultFragmentRendererContext"
 		],
 		"from": "setSegmentsExperienceIds(long[] paramName)",
-		"issueKey": "LPS-198401",
+		"issueKey": "layoutItemSelectorCriterion.setShowHiddenPages(true);",
 		"to": ""
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:815]: @kyle-miho this doesn't seem right. Is it a typo?

**Review by nicolas-sss — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json (hunk 1 above)
Comment (kyle-miho) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json:815]: oh oops it is, fixing it real quick @nicolas-sss !

Comment (kyle-miho) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json]: @nicolas-sss Fixed!

**Review by drewbrokke — APPROVED** [review]

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:hotel

**Comment by kyle-miho** [comment]
ci:test:relevant

**Comment by nicolas-sss** [comment]
LGTM 👍🏻

**Review by nicolas-sss — APPROVED** [review]

**Comment by kyle-miho** [comment]
ci:forward:force

**Pull request closed without merging**